shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -1272,6 +1272,104 @@ \subsection{Chain Transition}
   \label{fig:rules:chain}
 \end{figure}
 
+\clearpage
+
+\subsection{Byron to Shelley Transition}
+\label{sec:byron-to-shelley}
+
+This section describes how to transition the state held by the Byron ledger to the Shelley ledger.
+The Byron ledger state $\CEState$ is defined in \cite{byron_chain_spec}.
+Figure~\ref{fig:functions:to-shelley} defines a function $\fun{toShelley}$
+which takes an initial nonce $\eta$ and the Byron ledger state,
+and creates the Shelley ledger state.
+
+%%
+%% Figure - Byron to Shelley State Transition
+%%
+\begin{figure}[htb]
+  \emph{Applying a reward update}
+  %
+  \begin{align*}
+      & \fun{toShelley} \in \Seed \to \CEState \to \ChainState \\
+      & \fun{toShelley}~\eta~
+      \left(
+        \begin{array}{c}
+          \var{s_{last}} \\
+          \var{sgs} \\
+          \var{h} \\
+          \var{utxo} \\
+          \var{ds} \\
+          \var{us}
+        \end{array}
+      \right)
+      =
+      \left(
+        \begin{array}{c}
+          (\eta,~\eta,~\eta) \\
```

Comment:
I am waiting to hear back from the cryptographers, but it sounds like it might be preferable to not hard code a value in advance. One suggestion was to let `\eta` be the hash of the Byron `utxo`.